openrtb_ext/request_wrapper_test.go

```diff
@@ -2344,6 +2344,20 @@ func TestRegExtUnmarshal(t *testing.T) {
 			expectGPC:   ptrutil.ToPtr("some_value"),
 			expectError: false,
 		},
+		{
+			name:        `valid_gpc_json "1"`,
```

Comment:
Nitpick: The Go test runner replaces spaces in test names with an underscore. To make it easier to search for tests by name, please do not use spaces. Consider renaming to  `valid_gpc_json_"1"` and `valid_gpc_json_1`,

Reply:
DONE